.bazelrc

```diff
@@ -5,11 +5,18 @@ startup --host_jvm_args="-Xmx8g"
 build --announce_rc
 build --color=yes
 
-build:production --config=lsan --strip=never --copt=-O3
+build:production --config=lsan --copt=-O3
 
 # C/C++ CONFIGS
 build --cxxopt=-std=c++14
-build --compilation_mode=dbg
+# Create debug information only for magma binaries (not for external dependencies).
+# --compilation_mode=dbg would also create debug information of external dependencies 
+# and increase the size of artifacts drastically.
+# Needs --strip=never so that debug information is not removed by the linker.
+# See https://bazel.build/docs/user-manual#compilation-mode and
+# https://bazel.build/docs/user-manual#strip
+build --strip=never
+build --per_file_copt=^lte/gateway/c/.*$@-g
```

Comment:
Shouldn't the `compilation_mode` already set the required flags?

Reply:
yes, but then -g is used also for external dependencies - which drastically increases the c/cpp artifacts

Reply:
or did I misunderstood your point? you cannot use `compilation_mode` in `--per_file_copt` if this was your question.

Reply:
Ah no, I just overlooked the removal of the compilation_mode.